generator/internal/rust/templates/crate/src/builders.rs.mustache

```diff
@@ -181,7 +181,8 @@ pub mod {{Codec.ModuleName}} {
         {{/InputType.OneOfs}}
     }
 
-    impl gax::options::RequestBuilder for {{Codec.BuilderName}} {
+    #[doc(hidden)]
```

Comment:
Seems like this could be a separate PR 🤷‍♂️

Reply:
> Seems like this could be a separate PR 🤷‍♂️

I could have done a separate PR for sealing the `RequestOptionsBuilder` trait:

https://github.com/googleapis/google-cloud-rust/blob/e1fe15d02aa40c3158c8479ee320323a01e53f60/src/gax/src/options/mod.rs#L166

the rest of the PR is about hiding `RequestBuilder` (from the docs)

:shrug: